packages/core/utils/src/convert-query-params.ts

```diff
@@ -610,7 +610,7 @@ const transformParamsToQuery = (uid: string, params: Params): Query => {
 
   const query: Query = {};
 
-  const { _q, sort, filters, fields, populate, page, pageSize, start, limit } = params;
+  const { _q, sort, filters, fields, populate, page, pageSize, start, limit, ...rest } = params;
```

Comment:
which other params can be sent ?

Reply:
it's to allow chaining with the other functions (data, lookup at the moment)